### ISSUE TITLE
Update Sysbox package installer to support versions beyond Ubuntu Focal (20.04)

### ISSUE THREAD

**Issue by ScottG489** [nestybox/sysbox#411]
Referenced in the [Sysbox Distro Compatibility documentation](https://github.com/nestybox/sysbox/blob/master/docs/distro-compat.md), there is no package installer available for Ubuntu versions beyond Focal (20.04).

I know it's possible to [build and install from source](https://github.com/nestybox/sysbox/blob/master/docs/developers-guide/build.md#building--installing-from-source), but this would be very convenient.

**Comment by rodnymolina** [comment]
@ScottG489, that's right, Ubuntu's non-LTS releases are hard to track due to their short life-cycle, that's why we are only focusing on the LTS ones. However, you should be able to install the existing Ubuntu-Focal's package into any of those. Please let us know if you see any issue there.

We may need to include a comment about this in our documentation: non-LTS releases are not officially supported in the sense that we're not allocating resources to test them all, but they should certainly work.

**Comment by ScottG489** [comment]
> However, you should be able to install the existing Ubuntu-Focal's package into any of those. Please let us know if you see any issue there.

I believe I had tried this in the past, but it complained that I wasn't on the right version of Ubuntu and exited. Let me give it another shot and I'll report back here.
